core/src/test/java/org/jdbi/v3/core/statement/TestHashPrefixSqlParser.java

```diff
@@ -67,9 +66,10 @@ public void testBacktickOkay() {
     }
 
     @Test
-    public void testBailsOutOnInvalidInput() {
-        assertThatThrownBy(() -> parser.parse("select * from something\n where id = #\u0087\u008e\u0092\u0097\u009c", ctx))
-            .isInstanceOf(UnableToCreateStatementException.class);
+    public void testHashWithoutNameNotAParameter() {
+        String sql = "select * from something\n where id = #\u0087\u008e\u0092\u0097\u009c";
```

Comment:
same as before, I don't see how whatever this unicode string means is not a named parameter

Reply:
Parameter names are restricted to Java-ish parameter names:

```antlr
fragment NAME: 'a'..'z' | 'A'..'Z' | '0'..'9' | '_' | '.' | '?.' | '#';
```